package.json

```diff
@@ -28,22 +34,28 @@
   },
   "homepage": "https://github.com/SortableJS/angular-sortablejs#readme",
   "dependencies": {
-    "@angular/common": "^2.0.0",
-    "@angular/core": "^2.0.0",
-    "@angular/forms": "^2.0.0",
-    "rxjs": "^5.0.0-beta.12",
+    "@angular/common": "2.1.0",
```

Comment:
yes, typescript is not a problem at all, just was curious about its version :) Still I think users might decide on what version they use. The library is way too small to see the differences between the minor versions, so I think I'll return it to the previous version...